packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -19,7 +32,13 @@ const componentName: string = "stark-dropdown";
 		class: componentName
 	}
 })
-export class StarkDropdownComponent implements OnInit, OnChanges, OnInit {
+export class StarkDropdownComponent extends StarkAbstractColor implements OnInit, OnChanges, OnInit {
+	/**
+	 * Color theme of the dropdown
+	 */
+	@Input()
+	public color?: string;
```

Comment:
There is no need to re-declare the `color` property here, otherwise there is no benefit of having an abstract ;)